tools/ci_build/github/azure-pipelines/templates/c-api-cpu.yml

```diff
@@ -82,11 +82,13 @@ stages:
       packageName: 'onnxruntime-android'
       ReleaseVersionSuffix: $(ReleaseVersionSuffix)
 
+- stage: Final_AAR_Testing_Android_Training_Full
+  dependsOn: Android_Java_API_AAR_Packaging_Training_Full
```

Comment:
Move the dependency to Stage level, so we can better tract it with find in the editor.